web/app/services/build-log-websocket.service.spec.ts

```diff
@@ -12,6 +13,7 @@ describe('BuildLogWebsocketService', () => {
       providers: [
         BuildLogWebsocketService,
         {provide: DOCUMENT, useValue: {location: {host: 'host'}}},
+        {provide: AuthService, useValue: new AuthService(null)},
```

Comment:
It's best if you mock out the `.token` method of the authService and use that as the value instead.

ex. https://github.com/fastlane/ci/blob/master/web/app/login/login.component.spec.ts#L28